html/rendering/non-replaced-elements/the-fieldset-element-0/legend-align.html

```diff
@@ -0,0 +1,46 @@
+<!doctype html>
+<title>legend align</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<fieldset><legend align=left>x</legend></fieldset>
+<fieldset><legend align=center>x</legend></fieldset>
+<fieldset><legend align=right>x</legend></fieldset>
+<div align=left>
+ <fieldset><legend>x</legend></fieldset>
+</div>
+<div align=center>
+ <fieldset><legend>x</legend></fieldset>
+</div>
+<div align=right>
+ <fieldset><legend>x</legend></fieldset>
+</div>
+<div style="text-align: center">
```

Comment:
It doesn't seem right to me that text-align should affect the position of the legend block. It should only affect alignment of the text itself within lines.

Reply:
I guess we have a choice here. WebKit and Chromium position based on text-align, EdgeHTML and Gecko do not.

Reply:
What WebKit / Chromium do should be considered a bug.

Reply:
Wait, doesn't Gecko also do so with a special text-align value?

Reply:
Gecko has a special text-align value for `<div align>` and `<table align>` etc, but, AIUI, not for `<legend align>`.

Setting `text-align: center` on a table does not center the table with auto margins. But setting `text-align: center` positions a legend in the center, which is what @mstensho thinks is a bug.

Reply:
@bzbarsky could you double check the above?